augur/application/schema/alembic/versions/4_explorer_materialized_view_update.py

```diff
@@ -0,0 +1,656 @@
+"""Augur New Changes
+
+Revision ID: 4
+Revises: 3
+Create Date: 2023-01-24 09:17:31.706564
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql import text
+from augur.tasks.util.AugurUUID import AugurUUID, GithubUUID, UnresolvableUUID
+
+
+# revision identifiers, used by Alembic.
+revision = '4'
+down_revision = '3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
+    add_update_materialized_views_15()
+
+def downgrade():
+
+    upgrade=False
+
+    add_update_materialized_views_15(upgrade)
+
+def add_update_materialized_views_15(upgrade=True):
```

Comment:
This function isn't needed. You should just be able to call `add_materialized_views_15()` in the `upgrade()` and `downgrade()` functions

Reply:
@ABrain7710 : I made the change. Good catch!  Can you tell I was copying one of your scripts without completely understanding how it works? I am learning to manage the schema in Alembic. ;)